app/services/checksum_validator.rb

```diff
@@ -140,4 +145,30 @@ def latest_signature_catalog_entries
   def latest_moab_version
     moab_storage_object.version_list.last
   end
+
+  def validate_against_signature_catalog(data_content_file)
+    absent_from_manifest_data = { file_path: data_content_file, manifest_file_path: latest_signature_catalog_path }
+    paths_from_manifest = latest_signature_catalog_entries.map { |entry| signature_catalog_entry_path(entry) }
+    file_in_manifest = paths_from_manifest.any? { |entry| entry == data_content_file }
+    checksum_results.add_result(AuditResults::FILE_NOT_IN_MANIFEST, absent_from_manifest_data) unless file_in_manifest
+  end
+
+  def data_content_files
```

Comment:
I think this method is the right idea.  Two concerns:

1.  data/content   vs.   data/metedata
2.  nested subdirs under content   (need a test for this?  we ended up allowing this in MV, yes???)